python/demo/demo-part1.ipynb

```diff
@@ -158,7 +177,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ds.plot(d)"
+    "sc.plot(d)"
```

Comment:
Would it be possible to have a vertical axis label and units? Was that deliberately omitted?

Reply:
The plotting might go through a substantial re-write so I won't do anything about this now. It's a remnant from plotting multiple variables on the same plot, and sometimes they didn't have the same units, but I think when that's the case they different plots should be produced instead of having everything on the same axes